analyzer.py

```diff
@@ -33,53 +33,30 @@ def split_subtests(self):
 
         return split_data
 
-    def ttest(self):
-        """
-        Take the data and perform a cross-ttest on the rows.
-        Data returned looks like the following:
-        ```
-            [
-                {
-                    'ttest': 7.2,
-                    'pval': 0.01,
-                    'name1': macosx-raptor,
-                    'name2': macosx-browsertime
-                }, ...
-            ]
-        ```
-
-        :return dict: List of results.
-        """
-        results = []
+    def get_header(self):
+        template_header_path = "testing/resources/template/header"
+        with open(template_header_path, "r") as f:
+            template_header_content = f.read()
+            return template_header_content
 
-        split_data = self.split_subtests()
+    def get_analysis_template(self,func):
+        template_function_folder_path = "testing/resources/analysis-templates/"
+        template_function_file_path = template_function_folder_path + func
+        print(template_function_file_path)
+        with open(template_function_file_path,"r") as f:
+            return f.read()
 
-        for subtest in split_data:
-            done = {}
-            for entry1 in split_data[subtest]:
-                name = entry1["name"]
 
-                for entry2 in split_data[subtest]:
-                    if entry2["name"] == name:
-                        continue
-                    if (
-                        "%s-%s" % (name, entry2["name"]) in done
-                        or "%s-%s" % (entry2["name"], name) in done
-                    ):
-                        continue
-                    done["%s-%s" % (name, entry2["name"])] = True
+def main():
```

Comment:
Make sure you remove all of this `main` stuff when you're closer to being ready for landing.